frontend/public/components/cluster-settings/openid-idp-form.tsx

```diff
@@ -223,13 +224,13 @@ export class AddOpenIDPage extends PromiseComponent {
             required />
         </div>
         <div className="form-group">
-          <DroppableFileInput
-            onChange={this.clientSecretFileChanged}
-            inputFileData={clientSecretFileContent}
-            id="clientsecret-file"
-            label="Client secret"
-            isRequired
-            hideContents />
+          <label className="control-label co-required" htmlFor="clientSecret">Client Secret</label>
```

Comment:
@spadgett you're losing the drag/drop capability by changing this field, is that what you intended?

Reply:
@spadgett ah got it.